src/app/components/turnos/gestor-agendas/operaciones-agenda/suspender-agenda.component.ts

```diff
@@ -137,7 +137,7 @@ export class SuspenderAgendaComponent implements OnInit {
         this.smsService.enviarSms(smsParams).subscribe(
             sms => {
                 if (sms === '0') {
-                    this.plex.toast('info', 'Se envió SMS al paciente ' + turno.paciente.nombre + ' ' + turno.paciente.apellido);
+                    this.plex.toast('info', 'Se envió SMS al paciente ' + turno.paciente.alias || turno.paciente.nombre + ' ' + turno.paciente.apellido);
```

Comment:
Aca hay que poner el alias o nombre entre corchetes, sino te lo va a tomar como dos cadenas distintas y te var dar true la siempre en la primera antes del` ||`
```suggestion
                this.plex.toast('info', 'Se envió SMS al paciente ' + (paciente.alias || paciente.nombre) + ' ' + paciente.apellido);
```

Reply:
y aca el paciente se obtiene solo del turno, no existe como constante. Corregido lo de los paréntesis de todas formas :+1: